service/application/src/main/java/com/solace/maas/ep/common/model/ScanRequestDTO.java

```diff
@@ -23,11 +23,7 @@
 public class ScanRequestDTO extends BaseDTO {
 
     @NotNull(message = "Scan type must not be null")
```

Comment:
Should be scan types not scan type

Reply:
That's outside of the Scope of my PR. I didn't implement this class.